R/mergeData.R

```diff
@@ -0,0 +1,46 @@
+#' @title Incorporating feature engineering with clinical data
+#'
+#' @description Combines the feature engineering results from \code{computeRegionEntropy()}
+#' with clinical data to form a new data set.
+#'
+#' @details The function takes in three parameters: \code{feat_eng_df} is an output
+#' of \code{computeRegionEntropy()}. \code{test_clinical_data} is the clinical data
+#' that one wants to merge the entropy data with. \code{id_mapping} is an optional
+#' argument that allows the practitioner to map different forms of patient
+#' identification with each other. If this is \code{null}, then it will be assumed
+#' that the 'pid' column in \code{feat_eng_df} is the same identification as what
+#' is used in \code{test_clinical_data}.
+#'
+#' @param feat_eng_df A \code{tibble} that takes the same form as an output of
+#' \code{computeRegionEntropy()}.
+#' @param test_clinical_data A \code{dataframe} that includes the clinical data
+#' of the patients.
+#' @param id_mapping A \code{dataframe} that has columns in the following order:
```

Comment:
The data.frame should have these names but not necessarily in that order. We should adjust the code to reflect this (i.e no numeric indexing).

Reply:
I've replaced colnames() with rename() to fix this issue.